### PR TITLE
add --no-headers option for `kubectl config get-clusters`

### PR DESCRIPTION
**Pull request by WanLinghao**
**What this PR does / why we need it**:
add --no-headers option for `kubectl config get-clusters`
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
kubectl: add --no-headers option for `kubectl config get-clusters`
```

### CODE REVIEW

**Comment by neolit123** [comment]
@WanLinghao 

please add a release note:
```
kubectl: add --no-headers option for `kubectl config get-clusters`
```

/kind feature
/area kubectl 
@kubernetes/sig-cli-pr-reviews

**Comment by neolit123** [comment]
@kubernetes/sig-cli-maintainers 
/assign @droot 
/remove lifecycle-rotten

**Comment by idealhack** [comment]
this seems useful.

/reopen
/lifecycle frozen

@kubernetes/sig-cli-maintainers

**Comment by idealhack** [comment]
@WanLinghao could you reabse?

**Comment by WanLinghao** [comment]
@idealhack  rebase done

File: pkg/kubectl/cmd/config/get_clusters.go [hunk 1]
```diff
@@ -42,20 +42,27 @@ func NewCmdConfigGetClusters(out io.Writer, configAccess clientcmd.ConfigAccess)
...
 		Example: getClustersExample,
 		Run: func(cmd *cobra.Command, args []string) {
-			cmdutil.CheckErr(runGetClusters(out, configAccess))
+			noHeaders := cmdutil.GetFlagBool(cmd, "no-headers")
+			err := runGetClusters(out, configAccess, noHeaders)
+			cmdutil.CheckErr(err)
 		},
 	}
 
+	cmd.Flags().Bool("no-headers", false, "Don't print headers (default print headers).")
```
Comment (dixudx) [inline pkg/kubectl/cmd/config/get_clusters.go]: Cobra pflag will print the default value instead.

And the headers here are quite misleading.  I thought that were http headers.

**Review by dixudx — COMMENTED** [review]
What is this change for?

I think `config get-clusters | grep -v "NAME"` can do the same thing. Is it really necessary to make it in kubectl?

**Comment by idealhack** [comment]
@dixudx Hi Xu, thanks for the review. A `grep` can do this too, but `--no-headers` is a common flag in `kubectl`. I don't see a strong reason for excluding some subcommands.

File: pkg/kubectl/cmd/config/get_clusters.go (hunk 1 above)
Comment (soltysh) [reply pkg/kubectl/cmd/config/get_clusters.go:51]: Use consistent description as `kubecl get` has for `--no-headers` flag.

**Review by soltysh — CHANGES_REQUESTED** [review]
If you're going after this, I'd expect this being added to all `kubectl config` subcommands, not just this one.

**Pull request closed without merging**